src/express/server.js

```diff
@@ -174,7 +174,10 @@ class ExpressServer {
         const statusCode = _.get(routerStaticContext, 'statusCode', statusOK)
         if (!res.headersSent && statusCode < statusRedirect) {
           const idToken = _.get(req, 'cookies.id_token')
-          if (idToken) {
+          if (
+            idToken ||
+            globalEnv.releaseBranch === releaseBranchConsts.preview
```

Comment:
The logic here is getting complicate. I suggest we should extract the logic here to a separate function like `IsCacheable()`. Remove the comment `//not to cache personal response` below. Leave the comments to indicate in what situation the cache does not apply.